lib/rake/javaextensiontask.rb

```diff
@@ -208,6 +211,14 @@ def define_java_platform_tasks
       end
     end
 
+    def java_target_arg
+      if @release
+        "--release=#{@release}"
+      else
+        "-target #{@target_version} -source #{@source_version}"
```

Comment:
This will treated as a single argument.

How about this?

```ruby
      javac_command_line = [
          "javac",
          *java_target_args,
          java_lint_arg,
          "-d", tmp_path,
        ]
        # ...
      end
      # ...
      def java_target_args
        if @release
          ["--release=#{@release}"]
        else
          ["-target", @target_version, "-source", @source_version]
        end
      end

Reply:
Looks good!

Reply:
Pushed a commit to do so.